ci-operator/config/netobserv/network-observability-operator/netobserv-network-observability-operator-main.yaml

```diff
@@ -45,14 +45,44 @@ tests:
   optional: true
   steps:
     cluster_profile: aws
+    dependencies:
+      OO_INDEX: ci-index-noo-bundle
+    env:
+      OO_CHANNEL: latest
+      OO_INSTALL_NAMESPACE: openshift-netobserv-operator
+      OO_PACKAGE: netobserv-operator
+      OO_TARGET_NAMESPACES: '!all'
     test:
     - as: e2e-operator
-      commands: make test
+      cli: latest
+      commands: |
+        export PATH=$PATH:$HOME
+
+        if ! which kubectl; then
+          ln -s "$(which oc)" ${HOME}/kubectl
+        fi
+
+        export CSV=$(oc get csv -n openshift-netobserv-operator | grep -E "net.*observ" | awk '{print $1}')
+        USER=netobserv VERSION=main make set-agent-image
+        USER=netobserv VERSION=main make set-flp-image
+        # USER=netobserv VERSION=main make set-plugin-image
+        # make set-release-kind-downstream
+        PORT_FWD=false NAMESPACE=netobserv make deploy-loki
+        oc apply -f /go/src/github.com/netobserv/network-observability-operator/config/samples/flows_v1beta2_flowcollector.yaml
+        sleep 5
+
+        oc wait pods --timeout=120s --for=condition=ready -l app=flowlogs-pipeline -n netobserv
+        # oc wait -n netobserv --timeout=60s --for condition=Available=True deployment netobserv-plugin
+        oc wait pods --timeout=120s --for=condition=ready -l app=netobserv-ebpf-agent -n netobserv-privileged
```

Comment:
Something you could read here is simply the `Status` of the `FlowCollector` CR

Reply:
good idea, I will add that. thanks!